Source/Tests/TrackableEntities.EF.5.Tests/LoadRelatedEntitiesTests.cs

```diff
@@ -829,6 +830,27 @@ public void LoadRelatedEntities_Should_Populate_Multiple_Employees_Territories_W
 
 #endif
 
-        #endregion
-    }
+    #endregion
+
+        #region Complex Types
+
+        [Fact]
+        public void LoadRelatedEntities_With_Complex_Types()
+        {
+            var context = TestsHelper.CreateFamilyDbContext(CreateDbOptions.DropCreateDatabaseIfModelChanges);
+            var parent = new Parent(Guid.NewGuid().ToString());
+            parent.Address = new Address();
+            context.Parents.Add(parent);
+            context.SaveChanges();
+
+            // Act
+            context.LoadRelatedEntities(parent);
```

Comment:
OK great, I'll now commit a failing test for the async, then a fix for it with the following commit.